app/services/imports/import_report_service.rb

```diff
@@ -84,5 +84,52 @@ def generate_unassigned_logs_report(report_suffix)
 
       @logger.info("Unassigned logs report available in s3 import bucket at #{report_name}")
     end
+
+    def generate_missing_answers_report(report_suffix)
+      Rails.logger.info("Generating missing imported lettings logs answers report")
+      unanswered_question_counts = {}
+      missing_answers_example_sets = {}
+
+      LettingsLog.where.not(old_id: nil).where(status: "in_progress").each do |lettings_log|
+        applicable_questions = lettings_log.form.subsections.map { |s| s.applicable_questions(lettings_log).select { |q| q.enabled?(lettings_log) } }.flatten
+        unanswered_questions = applicable_questions.filter { |q| q.unanswered?(lettings_log) }.map(&:id) - lettings_log.optional_fields
```

Comment:
if we're always joining this by ", " let's just do that here instead of every time we refer to it?